wiki/Guides/Compressing_Files/en.md

```diff
@@ -7,110 +7,70 @@ outdated: true
 
 *This article was adapted from [a forum thread by ziin](https://osu.ppy.sh/community/forums/topics/60002).*
 
-Each osu! beatmapset has a file size limit dictated by its beatmaps' total lengths. In this tutorial I will tell you smart ways to get your beatmap under that limit.
+Each osu! beatmapset has a file size limit dictated by its beatmaps' total lengths. This guide will help you to get your beatmap under that limit.
 
 There are 2 types of compression. **Lossy** and **Lossless** compression.
 
 - **Lossless** implies that the quality never degrades and can be repeatedly compressed or decompressed.
 - **Lossy** is a much more powerful form of compression which sacrifices quality for space/processor demand.
 
-## Zip
-
-The `.osz` package is actually a `.zip` file, and will compress your beatmap, although not very much, since audio, video, and images are usually compressed very well. Zip and other archives are all lossless compression, otherwise you wouldn't be able to get your data back.
-
 ## Video
 
-Videos are usually where people go wrong. There are thousands of video codecs out there, and *believe it or not, YouTube is usually a terrible place to get videos*. For the best quality, you should always downsize from a high resolution source (720p YouTube videos are fine for this).
+Videos are usually where people go wrong. There are thousands of video codecs out there, and *believe it or not, YouTube is usually a terrible place to get videos*.
 
-h.264 and XviD are very good codecs to use but like most video codecs, they are lossy. So do not re-encode multiple times, but encode from the high resolution source.
+H.264 is a very good codec to use but like most video codecs, it is lossy. So do not re-encode multiple times, but encode from the high resolution source.
 
 ### Step-By-Step Guide
 
-*This section was adapted from [a forum thread by ziin](https://osu.ppy.sh/community/forums/topics/58235).*
-
-#### Precursor/Notes
-
-Editing videos with [avidemux](http://avidemux.sourceforge.net/download.html)
-
-#### Removing audio
+#### Removing audio from video
 
-1. Open avidemux
-2. Drag and drop the file into avidemux
-3. If the following shows up because the video is h.264 (h.264 is the best right now):
-   1. Press either (no is better unless the program crashes).
+This section will show you how to remove audio from videos using Handbrake to reduce the file size of the video file.
```

Comment:
```suggestion
This section will show you how to remove audio from videos using [Handbrake](https://handbrake.fr/) to reduce the file size of the video file.
```

Can you check if that is the right link to Handbreak?

Reply:
that is the correct link